sql/021-config-functions.sql

```diff
@@ -228,18 +228,25 @@ $$ LANGUAGE plpgsql;
 
 DROP FUNCTION IF EXISTS cs_encrypt_v1();
 
-CREATE FUNCTION cs_encrypt_v1()
+CREATE FUNCTION cs_encrypt_v1(force boolean DEFAULT false)
```

Comment:
Can we please get some documentation on what this force argument does, and how one might use it?